### PR TITLE
brewsci-superlu-dist 5.1.0 (new formula)

### PR DESCRIPTION
**Pull request by dpo**

### CODE REVIEW

**Comment by dpo** [comment]
Not sure why some tests fail. They don't fail locally.

File: Formula/brewsci-superlu-dist.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    doc.install "DOC/ug.pdf"
+    pkgshare.install "EXAMPLE"
+  end
+
+  test do
+    cp pkgshare/"EXAMPLE/dcreate_matrix.c", testpath
+    cp pkgshare/"EXAMPLE/pddrive.c", testpath
+    cp pkgshare/"EXAMPLE/g20.rua", testpath
+    system "mpicc", "-o", "pddrive", "pddrive.c", "dcreate_matrix.c", "-L#{Formula["brewsci-superlu-dist"].opt_lib}", "-lsuperlu_dist"
```
Comment (sjackman) [inline Formula/brewsci-superlu-dist.rb]:
```suggestion
    system "mpicc", "-o", "pddrive", "pddrive.c", "dcreate_matrix.c", "-I#{Formula["brewsci-superlu-dist"].opt_include}", "-L#{Formula["brewsci-superlu-dist"].opt_lib}", "-lsuperlu_dist"
```
Add `-I…/include`, and it should work.

**Comment by sjackman** [comment]
```
mpicc
-o
pddrive
pddrive.c
dcreate_matrix.c
-L/home/linuxbrew/.linuxbrew/opt/brewsci-superlu-dist/lib
-lsuperlu_dist

pddrive.c:25:27: fatal error: superlu_ddefs.h: No such file or directory
```
Add `-I…/include` and it should work.

**Comment by dpo** [comment]
Thanks @sjackman ! Next error:
```
==> brew readall --aliases brewsci/num
Error: No available formula with the name "brewsci/num/brewsci-superlu-dist" 
Exited with code 1
```

**Comment by dpo** [comment]
The command `brew readall --aliases brewsci/num` works locally (on macOS) and apparently works on linux, so I'm not sure what's going on.

**Comment by sjackman** [comment]
Try deleting this line…
https://github.com/brewsci/homebrew-num/blob/e93688d3d3443525c76d581f453a776a4f357b4c/.circleci/config.yml#L63

Try diffing `.circleci/config.yml` against the one in Brewsci/bio, and taking any changes from there. In particular, the above change, and using the `linuxbrew/brew` Docker image rather than `linuxbrew/linuxbrew`.

**Comment by sjackman** [comment]
💚

**Comment by dpo** [comment]
Phew!

**Pull request closed without merging**